config/flexdays.php

```diff
@@ -0,0 +1,12 @@
+<?php
```

Comment:
I want to confirm if the name of the file is ok? @drewroberts

Reply:
Let's make the name of the config file consistent with the package name. So, let's go with `config/flex-scheduling.php`